tests/trading/test_completeSets.py

```diff
@@ -68,7 +68,7 @@ def test_publicBuyCompleteSets_failure(contractsFixture):
 
     # buyCompleteSets exceptions
     with raises(TransactionFailed):
-        completeSets.publicBuyCompleteSets(market.address + 1, fxpAmount, sender=tester.k1)
+        completeSets.publicBuyCompleteSets(0, fxpAmount, sender=tester.k1)
```

Comment:
Did this start failing with `market.address + 1`?  I believe I intentionally didn't set it to `0` because I wanted to make sure it correctly failed if a non-special (but not a market) address was passed in.